scilpy/tractanalysis/features.py

```diff
@@ -45,6 +84,8 @@ def remove_loops_and_sharp_turns(streamlines,
         if tm.winding(s) < max_angle:
             ids.append(i)
             streamlines_clean.append(s)
+        else:
```

Comment:
I don't understand this else. This wont work (adding an empty single point streamlines).
Why is it there?